app/components/Incentive.tsx

```diff
@@ -7,20 +7,30 @@ import * as styles from "./Incentive.scss";
 
 export const HexagonText: React.SFC = () =>
   <div>
-    <p>Commitment Opportunity starts in:</p>
```

Comment:
There needs to be padding here, but should be fixed in a commit related to hexagon color changes